CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# dbt_ad_reporting v1.NEXT.RELEASE
```

Comment:
```suggestion
# dbt_ad_reporting vNEXTRELEASE
```
My only suggestion is to format the version without the dots so the auto releaser doesn't run.